File: java/src/com/company/NodeFromFile.java
package com.company;

public class NodeFromFile {
    private String nodeNumber;
    private String parentNodeNumber;
    private String childType;
    private String data;

    public NodeFromFile(String nodeNumber, String parentNodeNumber, String childType, String data) {
        this.nodeNumber = nodeNumber;
        this.parentNodeNumber = parentNodeNumber;
        this.childType = childType;
        this.data = data;
    }

    public String getNodeNumber() {
        return nodeNumber;
    }

    public void setNodeNumber(String nodeNumber) {
        this.nodeNumber = nodeNumber;
    }

    public String getParentNodeNumber() {
        return parentNodeNumber;
    }

    public void setParentNodeNumber(String parentNodeNumber) {
        this.parentNodeNumber = parentNodeNumber;
    }

    public String getChildType() {
        return childType;
    }

    public void setChildType(String childType) {
        this.childType = childType;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "{" +
               "nodeNumber:'" + nodeNumber + '\'' +
               ", parentNodeNumber:'" + parentNodeNumber + '\'' +
               ", childType:'" + childType + '\'' +
               ", data:'" + data + '\'' +
               '}';
    }
}

File: python/NodeFromFile.py
class NodeFromFile:
    def __init__(self, node_number, parent_node_number, child_type, data):
        self.node_number = node_number
        self.parent_node_number = parent_node_number
        self.child_type = child_type
        self.data = data

    @property
    def node_number(self):
        return self.__node_number

    @property
    def parent_node_number(self):
        return self.__parent_node_number

    @property
    def child_type(self):
        return self.__child_type

    @property
    def data(self):
        return self.__data

    @node_number.setter
    def node_number(self, node_number):
        self.__node_number = node_number

    @parent_node_number.setter
    def parent_node_number(self, parent_node_number):
        self.__parent_node_number = parent_node_number

    @child_type.setter
    def child_type(self, child_type):
        self.__child_type = child_type

    @data.setter
    def data(self, data):
        self.__data = data

    def __repr__(self):
        return "{" + "nodeNumber:{nodeNumber}, parentNodeNumber:{parentNodeNumber}, childType:{childType}, data:{data}".format(
            nodeNumber=self.node_number, parentNodeNumber=self.parent_node_number, childType=self.child_type,
            data=self.data) + "}"